apps/web/src/pages/ItTrainingFundPage/ItTrainingFundPage.tsx

```diff
@@ -0,0 +1,578 @@
+      />
+      <div data-h2-padding="base(x3, 0)">
+        <div data-h2-wrapper="base(center, large, x1) p-tablet(center, large, x2)">
+          <div
+            data-h2-display="base(flex)"
+            data-h2-flex-direction="base(column)"
+            data-h2-gap="base(x3)"
+          >
+            {/* Investing in the future of IT talent */}
+            <div>
+              <Heading
+                Icon={MapIcon}
+                size="h2"
+                color="primary"
+                data-h2-margin="base(0, 0, x1.5, 0)"
+              >
+                {intl.formatMessage({
+                  defaultMessage: "Investing in the future of IT talent",
+                  id: "v6/BRI",
+                  description:
+                    "Heading for section describing investing in future talent",
+                })}
+              </Heading>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "The Government of Canada is committed to supporting the development of its IT professionals. With the <strong>IT Community Training and Development Fund</strong>, employees represented by the Professional Institute of the Public Service of Canada (PIPSC) in the IT group now have increased access to a wide range of learning opportunities to build and deepen their IT skills.",
+                  id: "Sew1Zp",
+                  description:
+                    "First paragraph describing investing in future talent",
+                })}
+              </p>
+            </div>
+            {/* What is the IT Community Training and Development Fund? */}
+            <div
+              data-h2-display="base(flex)"
+              data-h2-flex-direction="base(column)"
+              data-h2-gap="base(x1.5)"
+            >
+              <Heading
+                Icon={BookmarkSquareIcon}
+                size="h2"
+                color="tertiary"
+                data-h2-margin="base(0)"
+              >
+                {intl.formatMessage({
+                  defaultMessage:
+                    "What is the IT Community Training and Development Fund?",
+                  id: "8p4ty0",
+                  description:
+                    "Heading for section describing the training fund",
+                })}
+              </Heading>
+              <p>
+                {intl.formatMessage(
+                  {
+                    defaultMessage:
+                      "The fund is a financial commitment to support the professional growth of the Government of Canada's IT staff. It was established under the <link>IT collective agreement</link> signed between the PIPSC IT group and the Treasury Board of Canada Secretariat in December 2023. The fund allocates $4.725 million each year for training and development for the duration of the agreement.",
+                    id: "A44nkj",
+                    description: "First paragraph describing the training fund",
+                  },
```

Comment:
Unfortunately there were a bunch of last minute changes to the copy (English and French).   There have probably been a few since you reviewed it.  Nienke has been reviewing the copy so you probably don't need to.